src/analysis/calculus/times_cont_diff.lean

```diff
@@ -1880,7 +1880,14 @@ begin
     exact (h i).zero_eq x hx },
   { intros m hm x hx,
     have := has_fderiv_within_at_pi.2 (λ i, (h i).fderiv_within m hm x hx),
-    convert (L m).has_fderiv_at.comp_has_fderiv_within_at x this },
+    -- `convert (L m).has_fderiv_at.comp_has_fderiv_within_at x this` no longer works.
+    convert has_fderiv_at.comp_has_fderiv_within_at x _ this,
+    rotate 2,
+    convert linear_isometry_equiv.has_fderiv_at _,
+    rotate 1,
+    convert L m using 1,
+    refl,
+    refl, },
```

Comment:
I can't work out what's going on here - `(L m).has_fderiv_at` is now a type error, but it's very difficult for me to work out which bits of the types mismatch.

Reply:
Maybe you already fixed it? When I change it back to check the error, I don't get an error.

Reply:
Are you sure you had the right cache loaded? I tested this in the last commit it was still broken.

Reply:
I am relatively sure. Maybe it's a lean 3.27 / lean 3.28 issue?
```text
% git status
On branch eric-wieser/smul_mem_of_tower
Your branch is up to date with 'mathlib/eric-wieser/smul_mem_of_tower'.
% git pull
Already up to date.
% leanproject get-cache
Looking for local mathlib oleans
Found local mathlib oleans
% lean --make src/analysis/calculus/times_cont_diff.lean 
% lean --version
Lean (version 3.27.0, commit de35266fe596, Release)
```

Reply:
I pushed commit 659086ab2 to a new branch, let's see what CI thinks.

Reply:
Seems to work? Perhaps I was running into another hash collision?

Reply:
The commit passed the build stage, so can you merge it back into your branch?